google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageRetryHandler.java

```diff
@@ -147,7 +148,7 @@ static boolean isRetryable(final StorageException exs) {
     // Yes one wouldn't necessarily think of 403 as retriable,
     // but see https://github.com/broadinstitute/gatk/issues/3735
     // for reports of 403 being returned in cases when retrying works.
-    return exs.isRetryable() || exs.getCode() == 403 || exs.getCode() == 500 || exs.getCode() == 503;
+    return exs.isRetryable() || exs.getCode() == 403 || exs.getCode() == 500 || exs.getCode() == 503 || exs.getCode() == 502;
```

Comment:
could you reformat these so they are in numerical order please

Reply:
Done